sklearn/kernel_approximation.py

```diff
@@ -38,7 +38,7 @@ class RBFSampler(TransformerMixin, BaseEstimator):
         Number of Monte Carlo samples per original feature.
         Equals the dimensionality of the computed feature space.
 
-    random_state : int, RandomState instance or None, optional (default=None)
```

Comment:
Could you revert all the changes in the `kernel_approximation` file. It is not related with your PR.